src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -6,9 +6,33 @@ description: Overview and specification of the hardware used by NUbots.
 slug: /projects/hardware/overview
 ---
 
-The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
+The NUbots team uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
 
-![Coronal image of the NUgus robot](../../../images/front-robot.png)
-![Sagittal image of the NUgus robot](../../../images/side-robot.png)
+![Coronal and saggital images of the NUgus robot](../../../images/nugus.png 'Coronal and saggital images of the NUgus robot')
 
-The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/). The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm). The robots use two [FLIR Blackfly S](https://flir.app.boxcn.net/s/x0r0bdsjstdkudjd2dwlxoa5fdvj2gpg/file/418603726810) cameras positioned in their head. In the torso lies the Intel NUC7i7bnh, 8GB RAM, 256GB SSD, Robotis CM740 subcontroller, and a 3850mAh 4s LiPo battery.
+## igus® Modifications
+
+The following modifications were made to fit our specific requirements:
+
+- The components of the robot were split to to fit the size of the printer bed and volume.
+- The torso was modified to have open shoulder covers for shoulder servo cable access.
+
+## Servos
+
+The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm), with two in each ankle, three in each hip, and one in each knee.
```

Comment:
Flywheel mounting brackets are custom made (milled from aluminium I think). Same as the bracket on the neck.

Cable clamps on the backs of the knees and on the ankle and hip blocks are 3D printed.

Reply:
servo to servo mounting brackets

Reply:
@tayloryoung6396 have you got CAD files for these parts?